Utilities/rXml.h

```diff
@@ -12,19 +14,21 @@ struct rXmlNode
 	std::string GetName();
 	std::string GetAttribute( const std::string &name);
 	std::string GetNodeContent();
+	void *AsVoidPtr();
 
-	void *handle;
+	pugi::xml_node *handle;
 	bool ownPtr;
 };
 
 struct rXmlDocument
 {
 	rXmlDocument();
-	rXmlDocument(const rXmlDocument& other);
-	rXmlDocument &operator=(const rXmlDocument& other);
+	//rXmlDocument(const rXmlDocument& other);
```

Comment:
Its a copy constructor so the code will still compile if you remove them, if a type shouldn't be copied then you need to make it explicit so it can't be done accidentally e.g  rXmlDocument(const rXmlDocument& other) = delete;